### ISSUE TITLE
"Brand" structured data is not being recognized by the Google Structured Data Tool

### ISSUE THREAD

**Issue by rmarcano** [Yoast/wpseo-woocommerce#334]
### What did you expect to happen?
The "Brand" structured data is expected to be recognized by Google. 

### What happened instead?
The Google Structured Data Tool is not recognizing it.

It seems that Google is not recognizing the "Brand" schema because the Microdata the Yoast WooCommerce SEO plugin is outputting is not correctly formatted. 

According to the [Google documentation](https://developers.google.com/search/docs/data-types/product), the microdata should have the following format (notice that the "name" apparently needs to be nested inside the "brand" itemprop.):
```
  <div itemprop="brand" itemtype="http://schema.org/Thing" itemscope>
      <meta itemprop="name" content="ACME" />
  </div>
```
The microdata the plugin outputs has the following format:
`<p itemprop="brand">ACME</p>`
 
In this case, the "name" is not nested inside the "brand".

### How can we reproduce this behavior?
1. Add the "Brand" product attribute to a product. The WooCommerce SEO plugin will output it as Microdata to the page. 
2. Test the page with the Google Structured Data Tool. See that it's not recognized.

### Technical info
* WordPress version: 5.1.1
* Yoast SEO version: 10.0
* WooCommerce SEO version: 10.0

**Comment by rmarcano** [comment]
Related issue: https://github.com/Yoast/wpseo-woocommerce/issues/284

**Comment by rmarcano** [comment]
Please inform the customer of conversation # 483839 when this conversation has been closed.

**Comment by rmarcano** [comment]
Please inform the customer of conversation # 483399 when this conversation has been closed.

**Comment by rmarcano** [comment]
Please inform the customer of conversation # 484005 when this conversation has been closed.

**Comment by Pcosta88** [comment]
Please inform the customer of conversation # 485836 when this conversation has been closed.

**Comment by rmarcano** [comment]
Please inform the customer of conversation # 489459 when this conversation has been closed.

**Comment by michaelbriantina** [comment]
Please inform the customer of conversation # 491853 when this conversation has been closed.

**Comment by mayada-ibrahim** [comment]
Please inform the customer of conversation # 492562 when this conversation has been closed.

**Comment by martijnvaneeghem** [comment]
fixed in 11.0
